examples/with-amp/pages/_document.js

```diff
@@ -0,0 +1,29 @@
+import Document, { Head } from 'next/document'
+import { DOMProperty } from 'react-dom/../lib/ReactInjection'
+
+DOMProperty.injectDOMPropertyConfig({
```

Comment:
Okay. Then that's totally fine.
Shall we add a few lines of comments?
After that I think we could take this.